manifests/concourse-manifest/extensions/datadog-agent.yml

```diff
@@ -1,8 +1,8 @@
 releases:
   - name: riemann
-    url: https://github.com/alphagov/paas-cf/releases/download/riemann-1/riemann-gds-1.tgz
-    sha1: 8a4220a039ce6a2a0383aab8d4a76746501e58ec
-    version: gds-1
+    url: https://s3-eu-west-1.amazonaws.com/gds-dev-boshreleases/riemann-gds-4.tgz
```

Comment:
Github now has a limit of 10MB per file for attaching files to releases. This release is ~150MB. We had to put it somewhere, so we created that bucket. We asked for feedback on the name but no one cared. Once we have a proper release build system in place, we can get rid of this temporary bucket & switch to a properly created release.